<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPM_MainWindow</class>
 <widget class="QMainWindow" name="DPM_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desktop Pattern Matcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="3">
     <widget class="QCheckBox" name="clickCheck">
      <property name="text">
       <string>click on matches</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="enableCheck">
      <property name="text">
       <string>enable</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="checkIntervalLabel">
      <property name="text">
       <string>check interval </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label">
      <property name="toolTip">
       <string>load of comparing thread</string>
      </property>
      <property name="text">
       <string>load: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="setImagePatternFromScreenShotButton">
      <property name="text">
       <string> Set Image From ScreenShot </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="checkIntervalValue">
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>10000000000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="loadLabel">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>load of comparing thread</string>
      </property>
      <property name="text">
       <string>--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QSpinBox" name="threshold2Value">
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QSpinBox" name="threshold1Value">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>threshold 1: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>threshold 2: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="10">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="patternGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>image pattern</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="patternLabel">
          <property name="text">
           <string>nothing to show</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="desktopCompareGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>desktop compare result</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: yellow;
background-color: rgb(100,100,100);</string>
          </property>
          <property name="text">
           <string> + potential matches </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: lime;
background-color: rgb(100,100,100);</string>
          </property>
          <property name="text">
           <string> + selected matches </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string> legend:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QLabel" name="desktopCompareLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>nothing to show</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="QGroupBox" name="tolerantMatchGroup">
      <property name="title">
       <string>tolerant match</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>threshold: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tolerantThresholdValue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
